operator_ui/src/components/Icons/Success.js

```diff
@@ -15,7 +21,7 @@ const Success = props => (
       </mask>
     </defs>
     <title>{'Artboard 1'}</title>
-    <circle cx={24} cy={24} r={22} fill="none" stroke="#fff" strokeWidth={4} />
+    <circle cx={24} cy={24} r={22} fill="none" stroke={isForStatusCard ? '#17c671' : "#fff"} strokeWidth={4} />
```

Comment:
I feel like we should pass the color in as a prop and default to `#fff` instead of coupling this to status cards